app/src/main/res/values/colors.xml

```diff
@@ -33,6 +33,6 @@
 
   <color name="transparent">#00000000</color>
 
-  <color name="facility_search_query_highlight">@color/simple_light_blue_100</color>
+  <color name="facility_search_query_highlight">#4DFFC800</color>
```

Comment:
The alpha value listed in the Figma file is 32%, the hex value for that is `52`, `4D` is 30% alpha. Can we make that change, please?